swarm/network/simulation/kademlia.go

```diff
@@ -96,3 +98,100 @@ func (s *Simulation) kademlias() (ks map[enode.ID]*network.Kademlia) {
 	}
 	return ks
 }
+
+func (s *Simulation) WaitTillSnapshotRecreated(ctx context.Context, snap simulations.Snapshot) error {
+	expected := listSnapshotConnections(snap.Conns)
```

Comment:
Please use prefix `get` not `list` ... we don't seem to use "list" for this behavior elsewhere:

`find $GOPATH/src/github.com/ethereum/go-ethereum/swarm/ -iname "*.go" -exec grep -P "list\S*\(" {} \;`